games-engines/frotz/patches/frotz-2.50.patchset

```diff
@@ -0,0 +1,67 @@
+From a07abb0b11fa6c3ceca559c8deec7e830db336a6 Mon Sep 17 00:00:00 2001
+From: Vrondir <nikiporumbachanov@gmail.com>
+Date: Thu, 12 Dec 2019 17:03:37 +0000
+Subject: Remove -pthread flag
+
+
+diff --git a/src/curses/Makefile b/src/curses/Makefile
+index 9f5ea43..8c9fe58 100644
+--- a/src/curses/Makefile
++++ b/src/curses/Makefile
+@@ -15,7 +15,7 @@ ARFLAGS = rc
+ SOUND_TYPE ?= ao
+ 
+ ifeq ($(SOUND_TYPE), ao)
+-  CFLAGS += -pthread
++#  CFLAGS += -lpthread
```

Comment:
There is a workaround for this, but this probably would be to much atm :)

Reply:
You can remove this for now, check the Makefile and set SOUND_TYPE to none with a patch